nano/templates/shuttle_manipulator.tmpl

```diff
@@ -0,0 +1,120 @@
+<!--
+Title: Shuttle Manipulator UI
+-->
+
+<h3>Operations</h3>
+<div class="item">
+	{{:helper.link('Status', 'link', {'selectMenuKey' : 'stat'}, data.selectedMenuKey == 'stat' ? 'selected' : null)}}
+	{{:helper.link('Templates', 'link', {'selectMenuKey' : 'stemp'}, data.selectedMenuKey == 'stemp' ? 'selected' : null)}}
+	{{:helper.link('Modification', 'link', {'selectMenuKey' : 'mod'}, data.selectedMenuKey == 'mod' ? 'selected' : null)}}
+</div>
+
+<div class="item">&nbsp;</div>
+
+<!-- Shuttle Status Tab -->
+{{if !data.selectedMenuKey || data.selectedMenuKey == 'stat'}}
+	{{for data.shuttles}}
+		<div class='statusDisplay'>
+			<div class='itemLabel'>
+			Name: {{:value.name}}
+			</div>
+			<div class='itemContentWide'>
+			ID: {{:value.id}}
+			</div>
+			<div class='itemContentWide'>
+			Shuttle Timer: {{:value.timeleft}}
+			</div>
+			<div class='itemContentWide'>
+			Shuttle Mode: {{:value.mode}}
+			</div>
+			<div class='itemContentWide'>
+			Shuttle Status: {{:value.status}}
+			</div>
+			<div class='itemContentWide'>
+			{{:helper.link('Jump to', 'space-shuttle', {'jump_to' : 1, 'type' : 'mobile', 'id' : value.id}, null)}}
+			{{:helper.link('Fast Travel', 'rocket', {'fast_travel' : 1, 'id' : value.id}, null)}}
+			</div>
+		</div>
+	{{/for}}
+<!-- Shuttle template tab -->
+{{else data.selectedMenuKey == 'stemp'}}
+	<div class="item">
+	{{for data.templates_tabs}}
+		{{:helper.link(value, 'space-shuttle', {'select_template_category' : value}, (data.existing_shuttle && data.existing_shuttle.id == value) ? 'selected' : null)}}
+	{{/for}}
+	</div>
+	{{if data.existing_shuttle}}
+		<div class="item">&nbsp;</div>
+		{{for data.templates[data.existing_shuttle.id].templates}}
+			<div class='statusDisplay'>
+				<div class='itemLabelWide'>
+					Name: {{:value.name}}
+				</div>
+				{{if value.description}}
+					<div class='itemContent'>
+						Description: {{:value.description}}
+					</div>
+				{{/if}}
+				{{if value.admin_notes}}
+					<div class='itemContent'>
+						Admin Notes: {{:value.admin_notes}}
+					</div>
+				{{/if}}
+
+				<div class='item'>
+					{{:helper.link('select template', 'space-shuttle', {'select_template' : 1, 'shuttle_id' : value.shuttle_id}, (data.existing_shuttle.shuttle_id == value.shuttle_id) ? 'selected' : null)}}
```

Comment:
Oh, oops, I didn't express myself clearly, whoops. What I *meant* to get across, is that:
`(data.existing_shuttle.shuttle_id == value.shuttle_id)`
should be replaced with:
`(data.selected && data.selected.shuttle_id == value.shuttle_id)`
leaving ` ? "selected" : null` intact - as that's the ternary operator that highlights the button when it is selected.